debs/SPECS/3.9.4/wazuh-api/debian/control

```diff
@@ -0,0 +1,12 @@
+Source: wazuh-api
+Section: admin
+Priority: extra
+Maintainer: Wazuh, Inc <info@wazuh.com>
+Build-Depends: debhelper (>= 7.0.50~), nodejs (>= 4.6), npm, curl, gawk
+Standards-Version: 3.8.4
+Homepage: http://www.wazuh.com
+
+Package: wazuh-api
+Architecture: any
+Depends: ${shlibs:Depends}, npm, nodejs (>= 4.6.1), wazuh-manager (>= 3.9.3), wazuh-manager (<< 3.9.4), curl
```

Comment:
This line must be updated to `wazuh-manager (>= 3.9.4), wazuh-manager (<< 3.9.5)`.

Reply:
https://github.com/wazuh/wazuh-packages/pull/228/commits/ca02cfa00eeb057c2e184ebf3ccea1922109ac1a